concent-vm/consts.yml

```diff
@@ -3,3 +3,6 @@ concent_dir:                 /home/vagrant/concent
 golem_dir:                   /home/vagrant/golem
 golem_hyperdrive_repository: https://github.com/mfranciszkiewicz/golem-hyperdrive/releases/download/v0.2.6/hyperg_0.2.6_linux-x64.tar.gz
 golem_hyperdrive_checksum:   sha256:94220ef8533030652d4a1c126f06fa9cdce9d620b91f1c4574a109a3891e8adf
+pyenv_repository:            https://github.com/pyenv/pyenv
+pyenv_dir:                   /opt/pyenv
+python_path:                 /opt/pyenv/versions/{{ specific_python_version_for_penv }}/bin/python{{ python_version }}
```

Comment:
One more thing. You're still using `specific_python_version_for_penv` here. And the name has a typo (should be `pyenv`).